metpy/calc/thermo.py

```diff
@@ -1205,3 +1205,43 @@ def _isen_iter(iter_log_p, isentlevs_nd, ka, a, b, pok):
 
     # output values as a list
     return ret
+
+
```

Comment:
Probably best to punt until we discuss in-person. I think we need to make sure we adhere to some kind of consistency regarding naming and the use of shortened forms in names.